frontend/packages/dev-console/integration-tests/utilities/appFunctions.ts

```diff
@@ -0,0 +1,71 @@
+import {click} from './elementInteractions';
+import {$, browser, ExpectedConditions as EC} from 'protractor';
+const waitForElement = 5000;
+
+export enum NavigationMenu {
+    Add = '+Add',
+    Topology = ' Topology',
+    Monitoring = 'Monitoring',
+    Builds = 'Builds',
+    Search = 'Search',
+    Helm = 'Helm',
+    ProjectDetails = 'Project Details',
+    ProjectAccess = 'Project Access'
+  }
+
+export const naviagteTo = async function(opt: NavigationMenu) {
```

Comment:
Isn't opt is of type string?

nit: typo.. needs to be changed in other places as well
```suggestion
export const navigateTo = async function(opt: NavigationMenu) {
```

Reply:
Ah, was in agreement with @karthikjeeyar until I saw that this is an enum. No, this is correct.

Calling it like `navigateTo(NavigationMenu)` fails:
![image](https://user-images.githubusercontent.com/8126518/77791641-6cdd9b80-703d-11ea-9168-da07c5809c5b.png)

Enums are not straight-passable... making a variable/property type-of enum, makes it one-of that enum.

Reply:
Thanks for the clarification andrew, so no change is required for this lines

Reply:
The name rename is still needed @makambalaji you have a typo.